### PR TITLE
[Add] Add MMC and Benchmark models

### PR DESCRIPTION
**Pull request by nishimoto**
I created article about MMC and Benchmark models. In addition, I unused pages in summary.md were deleted.

### CODE REVIEW

File: numerai-tournament/benchmark_models.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+## ベンチマークモデルの作成方法
+
+### ウォークフォワード交差検証
+
+すべての予測はウォークフォワードフレームワークを使用して行われます。これは、すべての予測が、予測が行われる日付より前に利用可能なデータのみを使用してトレーニングされたモデルを使用して行われることを意味します。
+
+具体的には、データは156Eraのチャンクに分割されます。それから、各Eraチャンクごとに、最初のEraチャンクからパージEraまでトレーニングされたモデルによって予測が行われます。パージEraの数は、20Dターゲットの場合は8であり、60Dターゲットの場合は16です。
+
+したがって、Era157から313はEra148までトレーニングされたモデルを使用して予測され、Era314-470は306までトレーニングされたモデルを使用して予測されます。以降も同様です。
```
Comment (ndharasz) [inline numerai-tournament/benchmark_models.md]: We have updated this section to be more clear:
https://docs.numer.ai/numerai-tournament/models#how-they-made

Could you update this section to match please?

**Review by ndharasz — CHANGES_REQUESTED** [review]
Thank you for submitting this PR - it looks great! We made an update to the Benchmark Model docs. Please update this sections to match and then I can approve.

File: numerai-tournament/benchmark_models.md (hunk 1 above)
Comment (nishimoto) [reply numerai-tournament/benchmark_models.md]: I added this following the updates in the English version. Thanks for your review!

**Comment by nishimoto** [comment]
I also updated `connect-index.md` because the Discord invite link was not working.

**Review by ndharasz — APPROVED** [review]
Looks great! Thank you for your contribution.

**Pull request merged**